File: 448.java
class Solution {
	public List<Integer> findDisappearedNumbers(int[] nums) {
		boolean[] seen = new boolean[nums.length + 1];
		seen[0] = true;
		for(int num : nums)
			seen[num] = true;	
		List<Integer> missing = new ArrayList<>();
		for(int i = 0; i < seen.length; i++) 
			if(!seen[i])
				missing.add(i); 
		return missing;
	}
}

File: 448.py
class Solution:
    def findDisappearedNumbers(self, nums):
        for val in nums:
            num = abs(val) - 1
            if nums[num] > 0:
                nums[num] *= -1
        return [i + 1 for i in range(len(nums)) if nums[i] > 0]
